archway-1/chain.json

```diff
@@ -212,6 +220,10 @@
       {
         "address": "archway-mainnet.grpc.l0vd.com:80",
         "provider": "L0vd.com"
+      },
+      {
+        "address": "https://archway-grpc.openbitlab.com",
```

Comment:
This is not valid GRPC endpoint address.
It needs to be `archway-grpc.openbitlab.com:443`

Also, I was not able to connect to that address:
```
grpcurl archway-grpc.openbitlab.com:443 list
Failed to dial target host "archway-grpc.openbitlab.com:443": context deadline exceeded
```

Reply:
Fixed